### PR TITLE
Evaluation

### PR DESCRIPTION
**Pull request by alex-hh**
* Integrate end-to-end evaluation into training pipeline.
* Refactor training script to make data and models easily configurable

### CODE REVIEW

**Comment by alex-hh** [comment]
n.b. there's a somewhat suspicious gap between train and val loss I'd like to clarify. It also seemed like train loss was affected by sorting of chain lists in src/data/cath_split_ids

**Review by JudeWells — APPROVED** [review]
Looks good!
